README.md

```diff
@@ -88,18 +88,35 @@ To set up the MVS, follow the steps below:
     
 ## Using the MVS
 
-To run the MVS with custom inputs, edit the json input file and run
+To run the MVS with custom inputs you have several options:
 
-    `python mvs_tool.py -i path_input_file -o path_output_folder`
+##### 1) Use the command line
 
-With `path_input_file`: path to json input file
+Edit the json input file (or csv files) and run
 
-and `path_output_folder`: path of the folder where simulation results should be stored
+    `python mvs_tool.py -i path_input_file -ext json -o path_output_folder`
+
+With 
+`path_input_file`: path to json input file,
+
+`ext`: json for using a json file and csv for using csv files
+
+and `path_output_folder`: path of the folder where simulation results should be stored.
 
 For more information about the possible command lines
 
 `python mvs_tool.py -h`
 
+##### 2) Use the `main()` function
+
+Edit the json (or csv files) and run the `main()` function. The following `kwargs` are possible:
```

Comment:
```suggestion
Edit the csv files (or, for devs, the json file) and run the `main()` function. The following `kwargs` are possible:
```

Reply:
What do you mean, "kwargs" are still possible? 
Is this the terminal command or something else? It might clash with #177, so please check it out.

Reply:
These `kwargs` can be given as input to the main func (can - no must!).

Reply:
Ah, okay. So eg. if this is included into another python tool.

Reply:
Right